.github/workflows/verify.yaml

```diff
@@ -58,4 +62,28 @@ jobs:
       - name: "Validate Terrform format"
         working-directory: mxd
         run: |-
-          terraform fmt -check -recursive  
+          terraform fmt -check -recursive
+          
+  End-To-End-Tests:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: password
+          POSTGRES_DB: backendservicedb
+        ports:
+          - 5432:5432
+    
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '21'
+          
+      - name: Change directory to backend-service and run tests
+        run: ./gradlew test
```

Comment:
Run `./gradlew clean build test`, please revalidate the command from tractusx repo. Code should be built as well, not just test. And if we have some checkstyle, those should also be executed.

Reply:
this is what I followed: [Connector Workflow](https://github.com/eclipse-edc/Connector/blob/4f3621811a8143167554206e1d9dbae2d60c9d23/.github/workflows/verify.yaml#L74)

this is how the connector configures E2E workflow